src/test/java/org/gridsuite/modification/server/service/ModificationRepositoryTest.java

```diff
@@ -456,16 +457,20 @@ public void insertModificationTest() {
         var groovyScriptEntity4 = GroovyScriptInfos.builder().script("script4").build().toEntity();
         var groovyScriptEntity5 = GroovyScriptInfos.builder().script("script5").build().toEntity();
         var groovyScriptEntity6 = GroovyScriptInfos.builder().script("scriptSaucisse").build().toEntity();
+        List<ModificationEntity> modificationEntities = List.of(groovyScriptEntity1, groovyScriptEntity2, groovyScriptEntity3, groovyScriptEntity4, groovyScriptEntity5, groovyScriptEntity6);
```

Comment:
That would nice to add a tabular modification with few elements to this list.
And then the test would show that the number of requests is not impacted by a tabular modification and its sub elelements.

Reply:
yes, done.
And it did not change the counters